src/accountant/core.cljs

```diff
@@ -9,7 +9,22 @@
            goog.history.Html5History
            goog.Uri))
 
-(defonce history (Html5History.))
+(defn- transformer-create-url
+  "Replacement createUrl TokenTransformer that avoids preserving the query string,
+  as Google's version incorrectly does. (See https://goo.gl/xwgUos)"
```

Comment:
I think there was previously an active open issue on this that this link went to. Perhaps we could find that issue again and check on its status (and update the link in this docstring, here)? At some point this will have been resolved.